jenkins/run-pytests

```diff
@@ -22,6 +21,9 @@ if [[ ${flake_sts} -ne 0 ]]; then
     printf -- "FAILED: flake8 --verbose .\n" >&2
 fi
 printf -- "\n"
+
+source ${progdir}/python-cache-cleanup.sh
+
```

Comment:
Certainly it is debatable if this would improve the performance.  At this point, performance is dominated by the pytests and legacy server tests.

Since this is working, such an investigation can be handled properly in another PR.

Reply:
Rather than change this change in a separate PR, I'd like to know why you inserted these three new lines into the _middle_ of this script instead of placing them _at the beginning_.

Reply:
I placed them before the tests.

Reply:
Yes, and I'm suggesting that they would be better placed a little farther before the tests (like at the beginning of this script or in the previous one).